### PR TITLE
Fix top zone

### PR DESCRIPTION
**Pull request by MatteoRobbiati**
In this PR:

- new home page name is: `Qibo | Home Page`;
- navbar dimension is equal to the qibocal reports one.

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -97,7 +97,7 @@ export default function Home() {
     <div className={styles.container}>
 
       <Head>
-        <title>Qibo Home Page</title>
+        <title>Qibo | Home Page</title>
```
Comment (AleCandido) [inline pages/index.js:100]: For consistency, we should put the `Qibo |` part in the global layout, and complete the rest with a variable to be set for each page.

However, we can also do this later on.

**Review by AleCandido — APPROVED** [review]

File: pages/index.js (hunk 1 above)
Comment (MatteoRobbiati) [reply pages/index.js:100]: I'm going to open a dedicated issue! Thank you.

**Review by Edoardo-Pedicillo — COMMENTED** [review]
Thanks @MatteoRobbiati, it looks good to me.

**Review by andrea-pasquale — COMMENTED** [review]
Thanks @MatteoRobbiati.
The navbar is bigger compared to that one of Qibocal. We should probably fix this.

**Comment by MatteoRobbiati** [comment]
> Thanks @MatteoRobbiati. The navbar is bigger compared to that one of Qibocal. We should probably fix this.

Yes, I think it is a problem related to the static definition of the navbar (on the other side the qibocal's one is built using bootstrap). I think that now it is fixed because I defined the height using pixels. 

Thank you very much for the suggestion.

**Pull request merged**